src/authn/ccloudProvider.ts

```diff
@@ -216,7 +216,11 @@ export class ConfluentCloudAuthProvider
       userInfo,
       session: undefined,
     });
-    (await getLaunchDarklyClient())?.identify({ key: userInfo.id });
+
+    const launchDarklyClient = await getLaunchDarklyClient();
+    if (launchDarklyClient) {
+      await launchDarklyClient.identify({ key: userInfo.id });
+    }
```

Comment:
We have to `await` getting the [client](https://github.com/confluentinc/vscode/blob/9df18cf77886ee669a75e298edc5ea507787cb8c/src/featureFlags/client.ts#L27), and `identify()` is an async method.